demosplan/DemosPlanCoreBundle/Entity/User/Customer.php

```diff
@@ -113,7 +113,7 @@ class Customer extends CoreEntity implements UuidEntityInterface, CustomerInterf
      *
      * @var ProcedureInterface
      *
-     * @ORM\OneToOne(targetEntity="demosplan\DemosPlanCoreBundle\Entity\Procedure\Procedure", mappedBy="customer")
```

Comment:
Why did you remove `mappedBy="customer"`?

Reply:
This was a bi-directional one-to-one relation.
Now there are two uni-directional one-to-one relations.
procedure -> holds reference to its customer
customer -> holds reference to its default-blueprint-procedure